client/blocks/app-banner/utils.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Internal dependencies
+ */
+import {
+	get,
+	includes,
+	reduce,
+} from 'lodash';
+
+export const PREFERENCE_NAME = 'appBannerDismissTimes';
+export const EDITOR = 'post-editor';
+export const NOTES = 'notifications';
+export const READER = 'reader';
+export const STATS = 'stats';
+export const ALLOWED_SECTIONS = [ EDITOR, NOTES, READER, STATS ];
+export const ONE_WEEK_IN_MILLISECONDS = 604800000;
+export const ONE_YEAR_IN_MILLISECONDS = 31540000000;
+
+export function getAppBannerData( translate, sectionName ) {
+	switch ( sectionName ) {
+		case EDITOR:
+			return {
+				title: translate( 'Rich mobile publishing.' ),
+				copy: translate( 'A streamlined editor with faster, simpler image uploading? Check and mate.' ),
+			};
+		case NOTES:
+			return {
+				title: translate( 'Watch engagement happening.' ),
+				copy: translate( 'Is your new post a hit? With push notifications, see reactions as they roll in.' ),
+			};
+		case READER:
+			return {
+				title: translate( 'Read online or off.' ),
+				copy: translate( 'Catch up with new posts when you have time, even if you are offline.' ),
+			};
+		case STATS:
+			return {
+				title: translate( 'Stats at your fingertips.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'WordPress at your fingertips.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).